packages/webapp/src/components/Animals/AddAnimalsFormCard/AddAnimalsFormCard.tsx

```diff
@@ -44,8 +45,7 @@ const FIELD_NAMES = {
   BATCH: 'batch',
 } as const;
 
-// Should be moved up to parent component that calls useForm
-type FormFields = {
+export type FormFields = {
```

Comment:
Sounds goods, and I assumed you meant import `FIELD_NAMES` back into this component (rather than import FIELD_NAMES _into_ types.ts) so I did that.

And I took this opportunity to also nest the folders 🙂